server/utils/nunjucksSetup.ts

```diff
@@ -24,9 +23,13 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
     })
   }
 
+  registerNunjucks(app)
+}
+
+export function registerNunjucks(app?: express.Express): Environment {
   const njkEnv = nunjucks.configure(
     [
-      path.join(__dirname, '../../server/views'),
+      'server/views',
```

Comment:
Had to juggle template paths around a fair bit to get thing working when both running normally and via tests.